plugins/dapustor/dapustor-nvme.c

```diff
@@ -526,7 +532,9 @@ static int dapustor_additional_smart_log(int argc, char **argv, struct command *
 	OPT_ARGS(opts) = {
 		OPT_UINT("namespace-id", 'n', &cfg.namespace_id, namespace),
 		OPT_FLAG("raw-binary",   'b', &cfg.raw_binary,   raw),
+#ifdef CONFIG_JSONC
 		OPT_FLAG("json",         'j', &cfg.json,         json),
+#endif /* CONFIG_JSONC */
```

Comment:
just thinking loud: if we introduce something like

```
#ifdef CONFIG_JSONC
#define OPT_FLAG_JSON OPT_FLAG("json",         'j', &cfg.json,       "Output In JSON format")
#else
#define OPT_FLAG_JSON 
#endif
```
on a global level we don't need these ugly `ifdefs` everywhere.

Reply:
Good suggestion. Fixed this by the PR #2637. Thank you.